client/src/Components/Forms/User/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import { Form, Formik } from 'formik'
+import Input from 'Components/Input/TextField'
+import Button from 'Components/Button'
+
+import * as Yup from 'yup'
+
+interface UserFormValues {
+  username: string
+  year: number
+}
+
+export interface UserProps {
+  /**
+   * Title of the form
+   */
+  title: string
+  /**
+   * Subimt Handler
+   */
+  onSubmit: (values: UserFormValues) => void
+  className?: string
+}
+
+const currentYear = new Date().getFullYear()
+
+const SignupSchema = Yup.object().shape({
+  username: Yup.string()
+    .trim()
+    .min(1, 'Username must be at least 1 character')
```

Comment:
This is unnecessary as the required field would fit the requirements of the min length